#goal definition
---
#result definition
float32 travel_time
---
#feedback
string status
float32 s_done
float32 s_tot0
float32 s_tot1
float32 s_tot2

